Fix:

error: call to undeclared function 'time'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]

https://sourceforge.net/p/vtwm/discussion/936866/thread/78f62f5b07/
--- add_window.c.orig	2018-04-21 00:39:31.000000000 -0500
+++ add_window.c	2024-08-08 01:30:41.000000000 -0500
@@ -38,6 +38,7 @@
 
 #include <stdio.h>
 #include <string.h>
+#include <time.h>
 #include "twm.h"
 #include <X11/Xatom.h>
 #ifndef NO_XPM_SUPPORT
